### PR TITLE
Outreach NL

### PR DESCRIPTION
**Pull request by Josketobben**
Dutch translation

### CODE REVIEW

**Comment by lh1008** [comment]
@Josketobben could you change the commits name. Be more specific, you can add the dutch language code and a small message describing your work.

**Comment by erciccione** [comment]
@Josketobben Thank you for recovering this. Did you use @MrGmv 's text from ? because if you did we need to give him some attribution adding him as co-author or in some other way.

**Comment by Josketobben** [comment]
@erciccione His contribution is negligible. His absence can be taken as a tacit admission of this.

**Comment by Josketobben** [comment]
@lh1008 oh jeez, more tech.. gimme 24h!

**Comment by erciccione** [comment]
Still, if you used his text entirely or partially he must be credited.

**Comment by Josketobben** [comment]
If he felt himself he contributed in any meaningful way he would be here, I reckon.

**Comment by Josketobben** [comment]
@lh1008 like this? I have no idea how to describe a Dutch translation other than to call it a Dutch translation.

**Comment by Josketobben** [comment]
Are you sure you're not creating a problem out of nothing? I think he's glad enough that I do believe he didn't just do a quick sloppy translation as a cash grab. It's quite unlikely to say the least that he'll come complain about being robbed from his priceless fruits of labor... And I could re-translate the paragraph or two (for which I used some of his elements just to be NICE) if the ridiculously unlikely were to happen anyway.

**Comment by erciccione** [comment]
I don't know if this is the first time you contribute to an open source project, but you need to understand that monero is a complex worldwide community with workflows and licensing problems to take care of, not a school project. What for you is a non-problem sets a dangerous precedent and is also against git's and versioning workflow in general. If you use somebody else work you **must** give them credit. A commit has legal value and you cannot just recommit somebody else's work and put your signature on it.

The problem would be solved by only adding on your PR "text partially translated by @mrgmv" or something like that, i don't understand why you refuse to give credits where are due.

**Comment by Josketobben** [comment]
I googled some words, shall we credit the dictionary and google as well? There's no denying language gets fuzzy in the details, so no need to use a condescending tone. But I do understand it's frustrating you have to deal with this linguistic black box and the unknown variables involved, and I'm sorry the project has taken this turn but please remember this isn't pleasant for me either. I would have preferred to walk in, review an almost finished translation, and be done with it. But that was not possible unless I cared neither about Monero nor my (future) role in it.

I've redone the relevant parts for your peace of mind.

**Comment by lh1008** [comment]
@Josketobben please read carefully what @erciccione says in his messages and the importance of licensing. You can read more here https://rfc.zeromq.org/spec:22/C4/. Mr_Gmv was not asking any type of reward so please don't talk about things you suppose. I had already told you this before, if you continue with this hard way to communicate with Monero contributors I will have to delete your PR and ask someone else to work the Dutch translation. Rudeness will not be tolerated in the Outreach repository.

**Comment by lh1008** [comment]
There was a misunderstanding of myself towards @Josketobben comments, I already spoke with him, so I deleted part of my comment. Good relationships are important. Thank you.

**Comment by erciccione** [comment]
Looks like the conflicting text was removed. We only need a review from a dutch speaker now.

**Comment by lh1008** [comment]
@erciccione I asked erin palmer to help us review the translation. I reminded him two days ago but seems he had been a little busy. Not sure if you know of anyone that could help us with this. You could let them know we have an active XMR reward for the contribution if they want.

**Comment by ProkhorZ** [comment]
Well, this is a weird situation. @MrGmv told me he tried to discuss his translation on IRC, but didn't get any response.

Having read both translations, I can't say that @Josketobben's version is significantly better. There's unusual terminology like 'blokketen' instead of 'blockchain' and Flemish turns of phrase like 'Moest...' and 'gesofistikeerde'. On the other hand, @MrGmv's translation is verbose and sometimes awkward. There are a few errors in both translations. I don't feel like analyzing them in detail at the moment.

Conclusions:
- It's not a problem to have more than one translation. One or both could be published on a personal website.
- Neither one is ready for publication, but unlike a previous attempt, they're good enough to be edited.

But at the moment, I have other priorities:
1. Documenting key terminology from the GUI wallet and my own translations.
2. Creating a 2-page flyer for the next [Monero Meetup in Utrecht](https://www.meetup.com/monero-meetup-utrecht-nl/) - even if that means a third translation.

File: monero-outreach-docs/translations/nl/PSA/files/forks.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Ken je hard forks
+
+Voor non-programmeurs kan de term "hard fork" verwarrende, zelfs negatieve connotaties dragen. Misschien heb je een paar artikels gelezen over "hard forks" en hoe dramatisch ze verdeling hebben gezaaid onder de cryptoprojecten. De "hard forks" van Monero nemen interessante vormen aan, en vallen ruwweg in drie categorieën:
+
+### 1) Netwerkupgrade / regelmatig geplande hark fork
```
Comment (ProkhorZ) [inline monero-outreach-docs/translations/nl/PSA/files/forks.md]: 'hark fork' should be 'hard fork'.

File: monero-outreach-docs/translations/nl/PSA/files/learning.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Door meer te leren over de functies, code, het onderliggend open-bron- en cypherpunkethos, en over hoe beveiliging en privacy te verbeteren tijdens gebruik, kunnen we samen Monero's ecosysteem versterken. Diep inzicht in alle aspecten van Monero helpt anderen erover aan te spreken en adoptie te versnellen. Je zal je comfortabeler en zekerder voelen in het bewaren, verzenden en ontvangen van XMR. Kennis is macht, en kan de wereld veranderen. Wees steeds leergierig.. en doe je eigen research!
+
+
+### De toekomst begint bij jezelf, hier volgt leesmateriaal om meer over Monero te leren:
+
+**_[getmonero.org (Officiele Website))](https://getmonero.org/)_**
```
Comment (ProkhorZ) [inline monero-outreach-docs/translations/nl/PSA/files/learning.md]: Start every line with a + to create bullet points. See the current source text.

File: monero-outreach-docs/translations/nl/PSA/files/dontshare.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+Misschien denk je dat het een goed idee is je privésleutels gewoon elders te bewaren. Maar wat als grootmoeder je instructies op de vensterbank laat liggen en een voorbijganger het noteert? Dan word je op een goede dag wakker met al je XMR verzonden naar een onbekend adres. Met Monero's privacy kan niet achterhalen wiens onbekende wallet het betreft, en zal je je XMR nooit meer terugkrijgen.
+
+Het kan ook voorkomen dat Monero's code gevorkt wordt tot een nieuw cryptoproject met de claim: 'vertienvoudig je bedrag in ruil voor je privésleutels'. Kom je dit tegen, blijf dan uit de buurt! Niet alleen probeert men je wellicht op te lichten, mogelijk wordt de hele integriteit van het Monero-netwerk ondermijnt. Oplichters hangen af van je hebzucht om hun schade te berokkeren.
+
+### Val niet ten prooi
+
+Het enige attribuut van de wallet dat je wil delen is je leessleutel - en zelfs dan ook enkel om inkomende transacties aan te tonen. Is dit niet nodig, deel dan geen sleutels. Als er meer wordt gevraagd dan je publieke walletadres (of soms je leessleutel), dan probeert men je waarschijnlijk op te lichten en te bestelen. Kortweg: deel nooit je privésleutels.
```
Comment (ProkhorZ) [inline monero-outreach-docs/translations/nl/PSA/files/dontshare.md]: 'Leessleutel' is a term we haven't been using so far, but we should discuss it. It's simpler and more natural than 'alleen-lezen sleutel'.

File: monero-outreach-docs/translations/nl/quickfacts/quickfacts.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
++ **_[Vermijd deze oplichters](https://www.reddit.com/r/Monero/wiki/avoid)_**
++ **_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
++ **_[Monero SWOT analyse]()_**
++ **_[de Monero gemeenschap](https://getmonero.org/community/hangouts/)_**
++ **_[Mastering Monero](https://masteringmonero.com/)_**
+
+**Monero Quick Facts - Gereviseerd op 14/11/2018**    
+**Geschreven voor de Monero-gemeenschap door Monero Outreach.**
+
+**Monero Outreach heeft als doel om adoptie en gebruik van Monero te stimuleren door middel van educatie en public relations. Jouw donaties maken dit mogelijk.**
```
Comment (ProkhorZ) [inline monero-outreach-docs/translations/nl/quickfacts/quickfacts.md]: This paragraph and the previous one are the same in @MrGmv's translation, so he should be credited.

**Comment by ProkhorZ** [comment]
The translation should also be updated due to #100

**Comment by erciccione** [comment]
> Well, this is a weird situation. @MrGmv told me he tried to discuss his translation on IRC, but didn't get any response.

Where on IRC, i constantly monitor the chat of the localization workgroup and i read nothing there.

> It's not a problem to have more than one translation. One or both could be published on a personal website.

It's really important to have consistency, so only one translation shall be added.

> Documenting key terminology from the GUI wallet and my own translations.

This is very appreciated, thanks.

**Comment by erciccione** [comment]
@Josketobben please add @ProkhorZ 's suggestion, then we can merge, the quality of the text can be improved later if necessary.

**Comment by lh1008** [comment]
@ProkhorZ thank you for your help, really appreciated. Agree with @erciccione, the quality can be improved later.

**Comment by Josketobben** [comment]
Alright, done.

I must say, @ProkhorZ, that's not a very fair summary. As opposed to filtering out bits of Flemish (thank you for spotting them), one can't fix verbosity and awkwardness with single-word substitutions. And these properties are often caused by literal translations, which the guidelines explicitly discourage. It's not even a subjective issue, it's just not possible to pretend both translations are comparable.

I indeed missed those two and a half sentences at the bottom, my apologies. They've been re-translated.

**Comment by ProkhorZ** [comment]
@Josketobben Thanks for the quick and detailed response, I appreciate that.

File: monero-outreach-docs/translations/nl/quickfacts/quickfacts.md [hunk 5]
```diff
@@ -51,47 +51,47 @@ Sinds de lancering heeft Monero verschillende aanzienlijke verbeteringen doorgem
 
 Het woord "Monero" komt van Esperanto. Men heeft deze taal gekozen omdat hij "gedecentraliseerd" is en symbool staat voor het breken van barrières tussen mensen wereldwijd. In Esperanto is "Monero" vrij opgebouwd uit drie elementen, elk één lettergreep: mon + er + o. Ieder heeft zijn eigen betekenis.
 
-**mon-:** geld
-
-**-er-:** kleinste deel
-
- **-o :** een ding / een zelfstandig naamwoord
++ **mon-:** geld
++
```
Comment (lh1008) [inline monero-outreach-docs/translations/nl/quickfacts/quickfacts.md]: @Josketobben, please remove the "+", this make it look like this:
spaces

Just leave it without the "+".

File: monero-outreach-docs/translations/nl/quickfacts/quickfacts.md [hunk 6]
```diff
@@ -123,11 +121,11 @@ bedrijven.
 
 + **_[Vermijd deze oplichters](https://www.reddit.com/r/Monero/wiki/avoid)_**
 + **_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
-+ **_[Monero SWOT analyse]()_**
 + **_[de Monero gemeenschap](https://getmonero.org/community/hangouts/)_**
 + **_[Mastering Monero](https://masteringmonero.com/)_**
++ **_[Satis Group verslag](https://research.bloomberg.com/pub/res/d37g1Q1hEhBkiRCu_ruMdMsbc0A)_**
 
-**Monero Quick Facts - Herzien op 14/11/2018**    
+**Monero Quick Facts - Herzien op 1/2/2018**
```
Comment (lh1008) [inline monero-outreach-docs/translations/nl/quickfacts/quickfacts.md]: *2019

File: monero-outreach-docs/translations/nl/quickfacts/quickfacts.md (hunk 5 above)
Comment (lh1008) [reply monero-outreach-docs/translations/nl/quickfacts/quickfacts.md]: Oh it was just made, thanks. Closing this comment.

**Comment by erciccione** [comment]
Ping @Josketobben

**Comment by Josketobben** [comment]
Right, the date fix. Done.

**Pull request merged**